core/corerepo/gc.go

```diff
@@ -23,10 +23,12 @@ var ErrMaxStorageExceeded = errors.New("maximum storage limit exceeded. Try to u
 type GC struct {
 	Node       *core.IpfsNode
 	Repo       repo.Repo
+	EnableGC   bool
```

Comment:
Why do we need this second `EnableGC`?

Reply:
Because I need to get this param in `func maybeRunGC(req *cmds.Request, node *core.IpfsNode) (<-chan error, error) `.